core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/components/dot-edit-page-state-controller/dot-edit-page-state-controller.component.ts

```diff
@@ -150,6 +205,36 @@ export class DotEditPageStateControllerComponent implements OnChanges {
         }
     }
 
+    /**
+     * Handle the click event on the dropdowns
+     *
+     * @param {{ event: PointerEvent; menuId: string }} { event, menuId }
+     * @memberof DotEditPageStateControllerSeoComponent
+     */
+    handleMenuOpen({ event, menuId }: { event: PointerEvent; menuId: string }): void {
+        this.menuOpenActions[menuId as DotPageMode](event);
+    }
+
+    /**
+     * Check if the dropdown button should be shown
+     *
+     * @private
+     * @param {string} mode
+     * @param {DotPageRenderState} pageState
+     * @return {*}  {boolean}
+     * @memberof DotEditPageStateControllerSeoComponent
+     */
+    private shouldShowDropdownButton(mode: DotPageMode, pageState: DotPageRenderState): boolean {
+        const shouldModeShowDropdown: Record<DotPageMode, boolean> = {
+            [DotPageMode.EDIT]:
+                this.featureFlagEditURLContentMapIsOn && Boolean(pageState.params.urlContentMap),
+            [DotPageMode.PREVIEW]: false, // Don't show for preview
+            [DotPageMode.LIVE]: false // Don't show for live
+        };
+
+        return shouldModeShowDropdown[mode];
+    }
```

Comment:
```suggestion
private static shouldModeShowDropdown: Record<DotPageMode, boolean> = {
    [DotPageMode.EDIT]: true,
    [DotPageMode.PREVIEW]: false,
    [DotPageMode.LIVE]: false
};

private shouldShowDropdownButton(mode: DotPageMode, pageState: DotPageRenderState): boolean {
    return (
        DotEditPageStateControllerSeoComponent.shouldModeShowDropdown[mode] &&
        this.featureFlagEditURLContentMapIsOn &&
        !!pageState.params.urlContentMap
    );
}
```

Reply:
This logics makes Preview and Live dependent of the feature flag logic. But I cleaned this code